chapters/stress/index.md

```diff
@@ -0,0 +1,130 @@
+# Jina Cloud Benchmark & Deployment
+
+[comment]: <> (TODO how do we sell this?)
+
+#### Purpose: Showcase how Jina scales and how you can deploy to AWS
+
+---
+
+#### Table of Contents
+
+  * [Infrastructure](#infrastructure)
+  * [Datasets](#datasets)
+    + [Image search](#image-search)
+    + [Text search](#text-search)
+  * [Configuration](#configuration)
+    + [Sharding parameters](#sharding-parameters)
+    + [Functional/Indexers parameter](#functional-indexers-parameter)
+    + [Client](#client)
+  * [Results](#results)
+  * [Test it yourself](#test-it-yourself)
+  * [Learnings](#learnings)
+
+## Infrastructure
+
+This example benchmark prioritizes a real-world scenario, where a user hosts Jina in the cloud. We use six machines in AWS, of instace size `c5.2xlarge` (8 vCPUs, 16GB RAM). We also add 100GB gp2 ssd.
```

Comment:
it will have to change for `indexer` which has 32 GB and 16 vCPUs

Reply:
Left a TODO about that inside the file